src/platforms/android/common/gradle.mdx

```diff
@@ -0,0 +1,240 @@
+---
+title: Gradle
+sidebar_order: 100
+description: "Learn about using the Sentry Android Gradle Plugin."
+redirect_from:
+  - /platforms/android/proguard/#gradle-configuration
```

Comment:
Unfortunately you can't redirect from a heading
```suggestion
```

Reply:
😿 @marandaneto should we redirect from `/platforms/android/proguard`? I don't think so, as the page still exists, but just checking. Also PTAL at the PR ;)

Reply:
If the page still exists, lets just remove the redirect and introduce the new page